lib/OfxParser/Ofx.php

```diff
@@ -170,8 +193,8 @@ private function buildBankAccount(SimpleXMLElement $xml)
 
         $bankAccount->statement = new Statement();
         $bankAccount->statement->currency = $xml->STMTRS->CURDEF;
-        $bankAccount->statement->startDate = $this->createDateTimeFromStr($xml->STMTRS->BANKTRANLIST->DTSTART);
-        $bankAccount->statement->endDate = $this->createDateTimeFromStr($xml->STMTRS->BANKTRANLIST->DTEND);
+        $bankAccount->statement->startDate = $this->createDateTimeFromStr($xml->STMTRS->BANKTRANLIST->DTSTART, true);
+        $bankAccount->statement->endDate = $this->createDateTimeFromStr($xml->STMTRS->BANKTRANLIST->DTEND, true);
```

Comment:
Same question as above for all changes to calls to `createDateTimeFromStr` - seems like this could be too permissive?